.github/workflows/test.yml

```diff
@@ -11,15 +11,33 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: '16'
+          node-version: "16"
 
       - name: Install dependencies
-        run: npm ci
+        run: |
+          npm i @nx/nx-linux-x64-gnu
+          npm ci
 
       - name: Install playwright browsers
         run: npx playwright install --with-deps
 
-      - name: Run unit tests
+      - name: Build project
+        run: npm run build
+
+      - name: Run unit tests for compas-open
```

Comment:
These test steps can be 1 step if you make use of lerna (npx lerna run test:unit)